kie-server-parent/kie-server-services/kie-server-services-common/src/main/java/org/kie/server/services/impl/marshal/MarshallerHelper.java

```diff
@@ -84,9 +83,23 @@ public String marshal(String marshallingFormat, Object entity) {
         	serverMarshallers.put(format, marshaller);
         }
 
-        return marshaller.marshall(entity, Collections.singletonMap(MARSHALLER_PARAMETER_STRICT, isStrictType(marshallingFormat)));
+        return marshaller.marshall(entity, buildParameters(marshallingFormat));
+    }
+    
+    
+    private  Map<String,Object> buildParameters(String contentType)
```

Comment:
reuse and set this in to the that class
https://github.com/kiegroup/droolsjbpm-integration/blob/56c0abc4a3d676ae24170642c05537577c32dad7/kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/marshalling/MarshallingFormat.java#L60